src/main/java/dev/devature/penguin_api/repository/AuthenticationRepository.java

```diff
@@ -0,0 +1,15 @@
+package dev.devature.penguin_api.repository;
+
+import dev.devature.penguin_api.entity.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AuthenticationRepository extends JpaRepository<User, Long> {
```

Comment:
Personally, I would go for using `UserRepository` for this. To me, a repository is something to encapsulate data access, but not really the data access associated to a single feature, especially something like authentication which is not really a concern of data access.

The data access here is something that I think would fit in a `UserRepository` without the need for a repository for authentication purposes

Reply:
I agree, this can use the existing `UserRepository` class in the main branch.